test/integration/mockiavelli.int.test.ts

```diff
@@ -151,6 +151,63 @@ describe(`Mockiavelli integration [${TEST_LIBRARY}]`, () => {
         expect(response.status).toEqual(200);
     });
 
+    test('wildcard matching - postfix', async () => {
+        ctx.mockiavelli.mock('/example/*', { status: 200 });
+        jest.spyOn(console, 'error').mockImplementation(() => {});
+
+        await expect(
+            ctx.makeRequest('GET', '/example/')
+        ).resolves.toMatchObject({ status: 200 });
+        await expect(
+            ctx.makeRequest('GET', '/example/foo')
+        ).resolves.toMatchObject({ status: 200 });
+        await expect(
+            ctx.makeRequest('GET', '/example/foo/bar.json')
+        ).resolves.toMatchObject({ status: 200 });
+        await expect(
+            ctx.makeRequest('GET', '/not-matching/')
+        ).resolves.toMatchObject({ status: 404 });
+    });
+
+    test('wildcard matching - prefix', async () => {
+        ctx.mockiavelli.mock('*/example/', { status: 200 });
+        jest.spyOn(console, 'error').mockImplementation(() => {});
+
+        await expect(
+            ctx.makeRequest('GET', '/example/')
+        ).resolves.toMatchObject({ status: 200 });
+        await expect(
+            ctx.makeRequest('GET', '/api/example/')
+        ).resolves.toMatchObject({ status: 200 });
+        await expect(
+            ctx.makeRequest('GET', '/v1/api/example/')
+        ).resolves.toMatchObject({ status: 200 });
+        await expect(
+            ctx.makeRequest('GET', '/not-matching/')
+        ).resolves.toMatchObject({ status: 404 });
+    });
+
+    test('wildcard matching - combined with query string', async () => {
```

Comment:
``/example?param=value`` is valid request. Is there a difference between ``/example?param=value`` and ``/example/?param=value`` for mockiavelli ?

Shouldn't we also add possibility to handle the above request if we are be able to handle ``/example`` request for ``'/example/*'`` ?

Reply:
I'm not sure how leading slash are handled. I'll create an issue to add tests and documentation around it.

Reply:
Done - #32